cognite/powerops/client/shop/data_classes/case.py

```diff
@@ -1,21 +1,18 @@
 from __future__ import annotations
 
 import logging
-import tempfile
+from collections.abc import Sequence as SequenceType
 from pathlib import Path
-from typing import Optional, TypedDict
 
 import yaml
+from typing_extensions import Self
 
-logger = logging.getLogger(__name__)
-
+from cognite.powerops.client.shop.data_classes.shop_file import SHOPFileReference, SHOPFileType
 
-class FileRefT(TypedDict):
-    file: str
-    encoding: str
+logger = logging.getLogger(__name__)
 
 
-class Case:
+class ShopCase:
```

Comment:
```suggestion
class SHOPCase:
```
Just for consistency in the code base. We've been uppercasing SHOP and other abbreviations so far.

Reply:
I actually had it like that in the code, but changed to to `ShopCase` to be consistent with `ShopRun`. We have both `SHOP` and `Shop` in the codebase, not ideal...

Reply:
Actually, NM! WE have `ShopRun` and `SHOPRun` in the code 😒